linalg/sparsemat.cpp

```diff
@@ -4120,10 +4121,12 @@ SparseMatrix * Add(real_t a, const SparseMatrix & A, real_t b,
    int * C_j;
    real_t * C_data;
 
+   A.HostReadI(), A.HostReadJ(), A.HostReadData();
    const int *A_i = A.GetI();
    const int *A_j = A.GetJ();
    const real_t *A_data = A.GetData();
 
+   B.HostReadI(), B.HostReadJ(), B.HostReadData();
```

Comment:
Ditto

Reply:
done in https://github.com/mfem/mfem/pull/4675/commits/ea041467ae1ad6d007ea2a6e7880fbfa448f50a7